src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -5,8 +5,36 @@ ifdef::cheri_standalone_spec[]
 WARNING: This chapter will appear in the unpriv spec after the RV32I chapter.
 endif::[]
 
-This chapter describes the {cheri_base32_ext_name} and {cheri_base64_ext_name} base capability instruction sets, that extend the RV32I and RV64I (or RV32E/RV64E) integer instruction sets with CHERI.
-If the hart supports RV32E/RV64E instead of RV32I/RV64I, {cheri_base_ext_name} the restrictions from <<rv32e>> apply and 16 general purpose registers instead of 32 -- but all other {cheri_base_ext_name} changes are identical so the remainder of this chapter only refers to extending RV32I/RV64I behavior.
+This chapter describes the {cheri_base32_ext_name} and {cheri_base64_ext_name} base capability instruction sets, that extend the RV32I and RV64I instruction sets with CHERI.
+Other standard options to bases are also supported such as E (16-registers), Zfinx and endianess.
+
+=== CHERI encoding formats
+
+CHERI implementations are required to choose a capability encoding format.
+The CHERI base ISA name is extended with the capability encoding format in use, and each implementation only supports one.
+
+The definition of all CHERI base ISAs include sufficient details of a capability encoding format to give enough details to allow execution of all base ISA instructions.
+
+As a result, the {cheri_base32_ext_name} and {cheri_base64_ext_name} names on their own do not describe complete base ISAs, as a capability encoding must be selected.
+
+The currently available base ISAs including capability encoding formats are listed below:
+
+* For {cheri_base64_ext_name}, the capability encoding format from the University of Cambridge covering the full 64-bit address space:
+** {rvy64_uni_base_name}
+** {rvy64e_uni_base}
+* For {cheri_base32_ext_name}, the capability encoding format from the University of Cambridge covering the full 32-bit address space and optional RISC-V compatibility with {cheri_default_ext_name}:
+** {rvy32_uni_base_name}
+ifndef::cheri_ratification_v1_only[]
+* For {cheri_base32_ext_name}E, the CHERIoT encoding covering 8MB of memory, optimized for smaller devices:
```

Comment:
Hm?  "covering 8MB of memory" is, I think, a mischaracterization of the CHERIoT v1 encoding; it can, and its root caps do, represent all 4GiB of the 32-bit address space, with 16MiB bounds alignment.  (And then, yes, it jumps down to at most 8MiB objects at 16KiB bounds alignment.)